cookbooks/calico/recipes/general_compute.rb

```diff
@@ -49,6 +51,20 @@
     sysctl net.ipv6.conf.eth0.forwarding=0
     EOH
 end
+ruby_block "persist-sysctl" do
```

Comment:
that might be what default.rb is for - but i don't want to spend time investigating just at the moment.  A lot of the compute/control nodes could be sorted out to share code